src/python/espressomd/analyze.pyx

```diff
@@ -32,12 +33,22 @@ from .utils import array_locked, is_valid_type
 from .utils cimport Vector3i, Vector3d, Vector9d
 from .utils cimport handle_errors, check_type_or_throw_except
 from .utils cimport create_nparray_from_double_array
-from .utils cimport create_nparray_from_double_span
 from .particle_data cimport get_n_part
 
 
-cdef _Observable_stat_to_dict(Observable_stat obs):
-    """Transform a scalar Observable_stat struct to a python dict.
+cdef _Observable_stat_to_dict(Observable_stat obs, int size,
+                              cbool calc_sp):
+    """Transform an Observable_stat struct to a python dict.
+
+    Parameters
+    ----------
+    obs :
+        Core observable.
+    size : :obj:`int`, \{1, 9\}
```

Comment:
This is already contained in `obs` right?

Reply:
yes, I just need to write a getter for the private member `m_chunk_size`

Reply:
Why do we store Coulomb energies in the particle slot? Shouldn't it be stored in the electrostatics solver slot, as it is done for dipolar energies?
https://github.com/espressomd/espresso/blob/35e18f7167f95f75057384993e0ef771bc061b12/src/core/cuda_interface.cpp#L204-L207

Reply:
I happen to know that it work if you change that 0 to 1. I think it's also more correct, because it is an aggregated term.

Reply:
I'll do the change for consistency with our slot convention, even if it changes nothing to the actual results (P3M energy is not implemented on GPU).